src/lib/storage/vector_compression/bitpacking/bitpacking_compressor.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "bitpacking_vector.hpp"
+#include "bitpacking_vector_type.hpp"
+#include "storage/vector_compression/base_vector_compressor.hpp"
+
+namespace opossum {
+
+class BitPackingCompressor : public BaseVectorCompressor {
+ public:
+  std::unique_ptr<const BaseCompressedVector> compress(const pmr_vector<uint32_t>& vector,
```

Comment:
Same question as above: why not `uint64_t`?

Reply:
because that was the existing interface of the `BaseVectorCompressor`. Do you want me to change that? But then we would need to implement fixed-width integer compression for uint64_t to support that. I can't think of a case where offset vector or attribute vector entries do not fit in that.

Reply:
No, that's what I meant with the review comment "ignore the uint32_t questions". When I finally realized that we are only using compact vectors for vector compression here, it all started to make sense. ;)